sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/AzMonListExtensions.cs

```diff
@@ -287,9 +287,9 @@ internal static string GetDefaultDbPort(string? dbSystem)
         }
 
         ///<summary>
-        /// Gets Database dependency target from activity tag objects.
+        /// Gets Database dependency target and name from activity tag objects.
         ///</summary>
-        internal static string? GetDbDependencyTarget(this AzMonList tagObjects)
+        internal static void GetDbDependencyTargetAndName(this AzMonList tagObjects, out string? dbTarget, out string? dbName)
```

Comment:
Do you think tuple return type would be best instead of out?

(string, string) GetDbDependencyTargetAndName(this AzMonList tagObjects)

Ref: https://stackoverflow.com/questions/748062/return-multiple-values-to-a-method-caller (second answer)

Reply:
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/out-parameter-modifier#:~:text=Declaring%20a%20method%20with%20out%20arguments%20is%20a%20classic%20workaround%20to%20return%20multiple%20values.%20Consider%20value%20tuples%20for%20similar%20scenarios.